cmd/get/volume.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"github.com/openebs/openebsctl/pkg/util"
+	"github.com/openebs/openebsctl/pkg/volume"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumesListCommandHelpText = `
+This command displays status of available zfs Volumes.
```

Comment:
This should be cstor volumes or this is generics so openebs volumes?

Reply:
This is supposed to be generic, I've updated it.